### PR TITLE
HS-1259 Rework code and add tests

### PR DESCRIPTION
**Pull request by chinh-cle**
## Jira link(s)
- https://issues.opennms.org/browse/HS-1259

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts

### CODE REVIEW

**Review by mikewrosey — APPROVED** [review]
Looks good

**Review by rcorry93 — APPROVED** [review]
LGTM, just added a comment for a possible change to the code.

File: ui/src/store/Mutations/alertsMutations.ts [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
-  // acknowledge alert
+  const { execute: acknowledgeAlerts, error: acknowledgeAlertsError } = useMutation(AcknowledgeAlertsDocument)
+
+  const { execute: clearAlerts, error: clearAlertsError } = useMutation(ClearAlertsDocument)
+
+  return {
+    acknowledgeAlerts,
+    acknowledgeAlertsError,
+    clearAlerts,
+    clearAlertsError
```
Comment (mikewrosey) [inline ui/src/store/Mutations/alertsMutations.ts:14]: If we can handle all errors in the gql service that would be best. Unless you need these for something specific.. doesn't look like they are used yet

**Review by mikewrosey — APPROVED** [review]

File: ui/src/store/Mutations/alertsMutations.ts (hunk 1 above)
Comment (chinh-cle) [reply ui/src/store/Mutations/alertsMutations.ts:14]: sure will look at it once I work on clear/acknowledge action

**Pull request merged**